src/components/vuestic-components/va-infinite-scroll/VaInfiniteScroll.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+  <component
+    :is="tag"
+    class="va-infinite-scroll"
+    :class="{'va-infinite-scroll--reversed': reverse}"
+  >
+    <slot
+      name="default"
+      ref="content"
+    />
+    <slot
+      name="loading"
+      v-if="!disabled"
+    >
+      <div ref="defaultSpinner"
+           class="va-infinite-scroll__spinner"
+           :class="{'va-infinite-scroll__spinner--invisible': !fetching}">
+        <va-progress-circle
```

Comment:
Progress circle still changes its size while spinning. You might want to fix this (sounds like 1-2 lines of code fix to me).